src/main/java/com/southernartsoap/model/CartDetails.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.southernartsoap.model;
+
+/**
+ *
+ * @author jared
+ */
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.Min;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+
+public class CartDetails {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "cart_details_id") //correct name?
+    private Long id;
+    
+    //product id witt ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY, optional = false) //shouldl this be optinal?
+    @JoinColumn(name = "product_id", nullable = false)//should this be nullable? //does this join to join column
+    private Product product;
+    
```

Comment:
"optional=false is a runtime instruction. The primary functional thing it does is related to Lazy Loading. You can't lazy load a non-collection mapped entity unless you remember to set optional=false (because Hibernate doesn't know if there should be a proxy there or a null, unless you tell it nulls are impossible, so it can generate a proxy.)
should not be nullable because we wouldn't want a line of the cart with no product in it. " -Affe, stackoverflow

Reply:
nullable should be false bc we wouldn't want to have a line in the cart with no product in it